build-logic/android/src/main/kotlin/convention.android-app.gradle.kts

```diff
@@ -9,6 +9,10 @@ plugins {
 android {
     testBuildType = "debug"
 
+    defaultConfig {
+        multiDexEnabled = true
```

Comment:
This solution doesn't solve the problem. We should decrease the method count in the framework (e.g. use `inline`s). Can discuss it later.

Reply:
Agree. Not sure that `inline` rescues =)